nixos/modules/services/networking/wireguard.nix

```
@@ -577,6 +577,7 @@ in
         }) all_peers;
 
     boot.extraModulePackages = optional (versionOlder kernel.kernel.version "5.6") kernel.wireguard;
+    boot.kernelModules = optional (config.security.lockKernelModules) "wireguard";
```

```suggestion
    boot.kernelModules = [ "wireguard" ];
```
Is there a downside to always loading the kernel module? As I understand it, it will be loaded as the service starts anyways..